### DISCUSSION TITLE
x64dbg中的白加黑利用 « 倾旋的博客

### DISCUSSION THREAD

**Discussion by giscus** [Rvn0xsy/rvn0xsy.github.io#3] [General]
# x64dbg中的白加黑利用 « 倾旋的博客

倾旋的博客

https://payloads.online/archivers/2022-08-17/1/

**Comment by Rvn0xsy** [comment]
这是一个评论测试！

**Comment by regomne** [comment]
杀毒软件只校验exe的签名，不校验被加载的dll的签名么？

**Reply by Rvn0xsy** [reply]
一般情况下很少会进行校验，具体可以使用物理机器去测试一下。

**Reply by injuryzy** [reply]
我最近也在弄这个，最好dll也要加签名，不然360直接爆了 ，

**Comment by injuryzy** [comment]
师傅，你github跳转文档链接有问题

**Reply by Rvn0xsy** [reply]
具体是哪个？
